modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, context, config, num_nodes, startup_timeout_sec, shutdown_tim
         self.init_logs_attribute()
 
         self.disconnected_nodes = []
+        self.start_ignite = kwargs.get("start_ignite")
```

Comment:
you should provide the default value for `start_ignite` - `kwargs.get("start_ignite", True)`